### PR TITLE
[FIX] website_event_filter_selector: Fix date selection

### PR DESCRIPTION
**Pull request by pedrobaeza**
cc @Yajo @antespi @rafaelbn

### CODE REVIEW

**Comment by rafaelbn** [comment]
Fixed :+1: thanks

**Comment by yajo** [comment]
:+1: thanks

File: website_event_filter_selector/views/templates.xml [hunk 1]
```diff
@@ -22,7 +22,7 @@
             <t t-foreach="dates" t-as="date">
                 <option
                     t-att-value="date[0]"
-                    t-esc="'%s (%s)' % (date[1], date[3])"
+                    t-esc="'%s (%s)' % (date[1], date[3]) if date[0] != 'old' else date[1]"
```
Comment (antespi) [inline website_event_filter_selector/views/templates.xml:25]: If date == 'old', then there is only one value for two %s places, isn't it?
Maybe with parenthesis it'll be clearer

```
t-esc="('%s (%s)' % (date[1], date[3])) if date[0] != 'old' else date[1]"
```

Comment (pedrobaeza) [inline website_event_filter_selector/views/templates.xml:25]: No, the expression is not analyzed that way. Operator precedence makes that you don't need parenthesis.

**Comment by yajo** [comment]
This fixes https://github.com/OCA/event/issues/51

File: website_event_filter_selector/views/templates.xml (hunk 1 above)
Comment (antespi) [inline website_event_filter_selector/views/templates.xml:25]: Well, that's why I point to add parenthesis, to be clearer.
But anyway, this is ok for me, :+1:

**Pull request merged**